### PR TITLE
Add hub

### PR DESCRIPTION
**Pull request by axelssonHakan**
- Added inline SVG plugin to be able to set the fill color on SVG
- Moved setting up the app (enigma) in app.jsx
- Render the "HUB" if global is possible to get but not the app
- Render QAE url input if gettining websocket error

### CODE REVIEW

**Comment by hrigner** [comment]
I think we should make AppProvider a function. Why the static props instead of AppProvider.propTypes?

**Comment by FredrikFolkesson** [comment]
Can I approve for the PR name only? 😍

File: package.json [hunk 1]
```diff
@@ -12,17 +12,18 @@
   },
   "dependencies": {
     "enigma.js": "2.3.0",
+    "node-sass": "4.9.4",
     "prop-types": "15.6.2",
     "react": "16.5.2",
     "react-dom": "16.5.2",
     "react-outside-click-handler": "1.2.2",
-    "node-sass": "4.9.4"
+    "react-svg-inline": "^2.1.1"
```
Comment (hrigner) [inline package.json]: Lock the version

File: src/components/app.jsx [hunk 2]
```diff
@@ -1,18 +1,121 @@
...
+
+  render() {
+    const {
+      app,
+      docs,
+      error,
+      engineURL,
+    } = this.state;
+
+    // Render the "HUB" (global is defined == connection to engine)
```
Comment (hrigner) [inline src/components/app.jsx]: The forbidden word... 🙀

File: package.json (hunk 1 above)
Comment (axelssonHakan) [reply package.json]: Done

File: src/components/app.jsx (hunk 2 above)
Comment (axelssonHakan) [reply src/components/app.jsx]: Missed it :) - will remove it then creating component of the **HUB**

**Pull request merged**